src/runtime/composables/useSwiper.client.ts

```diff
@@ -82,6 +83,7 @@ export function useSwiper(swiperContainerRef: Ref<SwiperContainer | null>, optio
       Object.assign(swiperContainerRef.value, options)
       swiperContainerRef.value?.initialize()
     }
+    swiper.value = swiperContainerRef?.value?.swiper
```

Comment:
It confused me，the value of 'swiperContainerRef?.value?.swiper' is reactive, but 'swiper.value' is not. You can resize the swiper and check the value of them.